container-registry/container-analysis/src/main/java/com/example/containeranalysis/Samples.java

```diff
@@ -320,30 +308,32 @@ public static int pubSub(String subId, int timeout, String projectId) throws Exc
 
     @Override
     public synchronized void receiveMessage(PubsubMessage message, AckReplyConsumer consumer) {
-      //every time a pubsub message comes in, print it and count it
+      // Every time a Pub/Sub message comes in, print it and count it
       System.out.println("Message " + messageCount + ": " + message.getData().toStringUtf8());
       messageCount += 1;
-      //acknowledge the message
+      // Acknowledge the message
       consumer.ack();
     }
   }
 
   /**
    * Creates and returns a pubsub subscription object listening to the occurrence topic
    * @param subId the identifier you want to associate with the subscription
-   * @param projectId the project's unique identifier
-   * @throws Exception on errors with the subscription client
+   * @param projectId the GCP project to create the subscription under
+   * @throws IOException thrown on errors with the subscription client
+   * @throws StatusRuntimeException if subscription already exists
+   *
    */
-  public static Subscription createOccurrenceSubscription(String subId, String projectId)
-      throws Exception {
+  public static Subscription createOccurrenceSubscription(String subId, String projectId) 
+      throws IOException, StatusRuntimeException {
+    // This topic id will automatically receive messages when Occurrences are added or modified
     String topicId = "resource-notes-occurrences-v1alpha1";
```

Comment:
@vtsao Is there a new topic Id for beta?

Reply:
Yup:
container-analysis-occurrences-v1beta1